### PR TITLE
feat: create settings screen component

### PR DESCRIPTION
**Pull request by Sandtory**
Added settingsscreen template.
Merged main into mad-screens.

### CODE REVIEW

**Comment by tormodAase** [comment]
This PR fails some checks. Let me know when those are resolved! Then I will start the review @Sandtory

File: packages/screens/src/SettingsTemplate/SettingsTemplate.tsx [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    | {
+          name: "switch";
+          isActive: boolean;
+          onChange: (isActive: boolean) => void;
+          switchSize?: "small" | "normal";
+          disabled?: boolean;
+          description?: string;
+          color?: string;
+      }
+    | { name: "custom"; param: any };
```
Comment (HaakonSvane) [inline packages/screens/src/SettingsTemplate/SettingsTemplate.tsx:25]: Lets have a discussion on what this custom type should provide after you are done looking at the other stuff in this PR

File: packages/screens/src/SettingsTemplate/SettingsTemplate.tsx [hunk 2]
```diff
@@ -0,0 +1,109 @@
+import React from "react";
+import { Button, Cell, Icon, IconName, Spacer, Typography } from "@equinor/mad-components";
+import { View, ScrollView } from "react-native";
+import { useNavigation } from "@react-navigation/native";
+
+export type CellType =
+    | {
+          name: "navigation";
+          param: { route: string; params?: Record<string, any> };
```
[the remark this answers is not in the document]
Comment (Sandtory) [reply packages/screens/src/SettingsTemplate/SettingsTemplate.tsx:9]: Great find!

[the remark this answers is not in the document]
Comment (Sandtory) [reply packages/screens/src/SettingsTemplate/SettingsTemplate.tsx:9]: thanks, the switch cell was a nice addition indeed😡

**Comment by Sandtory** [comment]
If my code was a burning building and you were a firefighter, consider my code extinguished 🔥🚒

**Comment by tormodAase** [comment]
@Sandtory should we close this? Since Settings Screen already has been added?

**Comment by Sandtory** [comment]
yes was thinking the same, can be closed @tormodAase

**Pull request closed without merging**